### PR TITLE
solution

### PR DESCRIPTION
**Pull request by serentinos**
- [DEMO LINK](https://serentinos.github.io/layout_calendar/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--mon calendar--month-length--30">
+      <div class="calendar__month">
```
Comment (andrusb21) [inline src/index.html]: i think it will be better to use semantic tags

**Review by andrusb21 — APPROVED** [review]
GJ

File: src/index.html (hunk 1 above)
Comment (serentinos) [reply src/index.html]: Thanks! I forgot about that.

File: src/index.html [hunk 2]
```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--mon calendar--month-length--30">
+      <ul class="calendar__month">
```
Comment (Anna-Tkachenko) [inline src/index.html]: Do you need extra wrapper here? Let's get rid of it, you've already have `div` as wrapper for the calendar

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

**Review by niaYaro — APPROVED** [review]
Great job! 🤟 

According to the task 'The hovered cell has to become pink', but you used another color. It's not critical. Just for your attention.

**Comment by serentinos** [comment]
> Great job! love_you_gesture
> 
> According to the task 'The hovered cell has to become pink', but you used another color. It's not critical. Just for your attention.

@niaYaro Just wanted to make it a little more unique :)

File: src/styles/utils/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+$field-width: ($day-cell-size + $cells-gap) * 7;
+
+$start-day: (
+  "mon": 0,
+  "tue": ($day-cell-size + $cells-gap),
+  "wed": ($day-cell-size + $cells-gap) * 2,
+  "thu": ($day-cell-size + $cells-gap) * 3,
+  "fri": ($day-cell-size + $cells-gap) * 4,
+  "sat": ($day-cell-size + $cells-gap) * 5,
+  "sun": ($day-cell-size + $cells-gap) * 6
```
Comment (Anna-Tkachenko) [inline src/styles/utils/_variables.scss:14]: you can move calculation in the cycle where you set the correct properties for each of this modifiers

**Review by Anna-Tkachenko — APPROVED** [review]
Good job!

**Pull request closed without merging**